tests/kat_kem.c

```diff
@@ -42,6 +43,19 @@ static void fprintBstr(FILE *fp, const char *S, const uint8_t *A, size_t L) {
 	fprintf(fp, "\n");
 }
 
+static int is_mceliece(const char *method_name) {
+	return ( !strcmp(method_name, OQS_KEM_alg_classic_mceliece_348864)
```

Comment:
Hmm -- this looks "tedious" and prone to future errors if McEliece changes names or parameters. Is there really no other common "McEliece" designator?